### PR TITLE
fix windows release by install sqlite3

### LINKED ISSUE #1157: [CI] Windows release failed
note: LINK : fatal error LNK1181: cannot open input file 'sqlite3.lib'

https://github.com/rooch-network/rooch/actions/runs/6900430147/job/18773561295

### PR DESCRIPTION
**Pull request by yourmoonlight**
## Summary

Summary about this PR

- Closes #1157

### CODE REVIEW

File: .github/workflows/release_asset.yml [hunk 1]
```diff
@@ -35,6 +35,24 @@ jobs:
...
+          call "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build\vcvars64.bat"
+          lib /machine:x64 /def:sqlite3.def /out:sqlite3.lib
+
+      - name: Set variables for sqlite (Windows)
+        if: matrix.platform == 'windows-latest'
+        shell: bash
+        run: |
+          echo "C:\ProgramData\chocolatey\lib\SQLite\tools" >> $GITHUB_PATH
+          echo "SQLITE3_LIB_DIR=C:\ProgramData\chocolatey\lib\SQLite\tools" >> $GITHUB_ENV
+          echo "SQLITE_DATABASE_URL=C:\test.db" >> $GITHUB_ENV
```
Comment (jolestar) [inline .github/workflows/release_asset.yml:54]: Is 'SQLITE_DATABASE_URL' env for the test?

Comment (yourmoonlight) [reply .github/workflows/release_asset.yml:54]: Yes, I'll remove it

Comment (yourmoonlight) [reply .github/workflows/release_asset.yml:54]: Yes, I'll remove it

File: .github/workflows/release_asset.yml [hunk 2]
```diff
@@ -35,6 +35,23 @@ jobs:
...
           platform: x64
 
+      #fix https://github.com/rooch-network/rooch/issues/1157  with https://github.com/diesel-rs/diesel/blob/master/.github/workflows/ci.yml
+      - name: Install sqlite (Windows)
+        if: matrix.platform == 'windows-latest'
+        shell: cmd
+        run: |
+          choco install sqlite
+          cd /D C:\ProgramData\chocolatey\lib\SQLite\tools
+          call "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build\vcvars64.bat"
```
Comment (baichuan3) [inline .github/workflows/release_asset.yml:45]: Does this require specifying the windows version?

Comment (yourmoonlight) [reply .github/workflows/release_asset.yml:45]: Yes, use windows-lastest should be ok.

Comment (baichuan3) [reply .github/workflows/release_asset.yml:45]: If Windows is upgraded, Microsoft Visual Studio\2022 will also be upgraded. For example, if it reaches Microsoft Visual Studio\2023, an error will be reported.

Comment (yourmoonlight) [reply .github/workflows/release_asset.yml]: > If Windows is upgraded, Microsoft Visual Studio\2022 will also be upgraded. For example, if it reaches Microsoft Visual Studio\2023, an error will be reported.

That makes sense, I'll try modifying it and see.

**Review by jolestar — APPROVED** [review]

**Pull request merged**